README.md

```diff
@@ -48,8 +48,12 @@ The figure above illustrates where we think GSP fits on a "PaaS Spectrum":
 
 ## Getting started
 
-- [Getting started with a GDS Supported Platform cluster](docs/gds-supported-platform/getting-started-gsp-cluster.md)
-- [Getting started with a local GDS Supported Platform development cluster](docs/gds-supported-platform/getting-started-gsp-local.md)
+Come talk to the Automate team. We're on Slack in #re-autom8 and on the east side of the 6th floor.
+
+You also might want to look at how other teams are using GSP:
+
+- [verify-proxy-node](https://github.com/alphagov/verify-proxy-node/tree/master/ci)
+- doc-checking
```

Comment:
```suggestion
- [Verify Document Checking Service (private repo)](https://github.com/alphagov/doc-checking)
```

Given these docs are internal, even though this repo is private I still think it's OK to link to them?

Reply:
yeah I wasn't sure which is why I just left it with no link. But you're right it doesn't make much of a difference.